andela/nazgul-rc
# #157244155 Integrate wallet

#### What does this PR do?
Integrates paystack payment as one of the payment options for users

#### Description of Task to be completed?
- Make a wallet schema in accout schema
- Include server methods to get balance of wallet

#### How should this be manually tested?

- npm run git clone `https://github.com/andela/nazgul-rc.git` to clone the repository
- navigate to the nazgul-rc folder
- run `curl https://install.meteor.com/ | sh`
- run `npm install -g reaction-cli`
- install dependencies with`meteor npm install`
- run `reaction` to start up server
- once app is running, navigate to localhost:3000
- navigate dropdown in mainpage and click on `wallet` to check wallet dashboard
- fund wallet by punching amount and hit submit button
- Navigate to checkout page and toggle on `pay with wallet`
- and make payment

#### What are the relevant pivotal tracker stories?
#157244155

#### Screenshots (if appropriate)
<img width="1440" alt="1" src="https://user-images.githubusercontent.com/29508652/40731665-74ed62e6-6429-11e8-8f14-92184e82c0aa.png">
<img width="1440" alt="2" src="https://user-images.githubusercontent.com/29508652/40731666-751541b2-6429-11e8-8f21-d78b8180f01a.png">
<img width="1440" alt="3" src="https://user-images.githubusercontent.com/29508652/40731667-753fbe2e-6429-11e8-8598-beaed57ff21f.png">
<img width="1440" alt="4" src="https://user-images.githubusercontent.com/29508652/40731753-a1923f38-6429-11e8-865a-34a8997f2b64.png">
<img width="1440" alt="5" src="https://user-images.githubusercontent.com/29508652/40731754-a1b85876-6429-11e8-88a7-00fd14a50d70.png">
<img width="1440" alt="6" src="https://user-images.githubusercontent.com/29508652/40731755-a1e39a68-6429-11e8-8885-4ebd4d403d5d.png">
